Hannah-Sten/TeXiFy-IDEA
# Fix line breaking of comments during formatting

Fix #1771

#### Summary of additions and changes

* Insert % sign in front of the next line when pressing enter inside a comment
* Includes magic comments and comment prefixes of multiple %

#### How to test this pull request

See tests. You can press enter in a comment, but it also works when formatting:

Enable in code style settings 'Ensure right margin is not exceeded', and set margin to 50.

```latex
% Calculated protections are able to develop anxious insurances when they prick notes and relate identities and rejects.
\begin{document}
%    This is valiantly boundless. This relation contains the blood cells for the anwser.
    This mobile quality is considered every 83 locations and is insulted on the white youth it took to call one performance.
    % These emails are to be determined by the science according to their vengeful deals.
    Instead accepting related between driving algorithms hard to pretend whether various shirts have helped.
\end{document}
```
